src/main/java/org/sonarsource/pluginpackaging/AbstractSonarMojo.java

```diff
@@ -105,9 +105,12 @@ public abstract class AbstractSonarMojo extends AbstractMojo {
   @Parameter(property = "sonar.pluginName", defaultValue = "${project.name}", required = true)
   private String pluginName;
 
-  @Parameter()
+  @Parameter
   private String sonarQubeMinVersion;
```

Comment:
My understanding reading `https://maven.apache.org/developers/mojo-api-specification.html` is that we could add `@Deprecated`. See [this](https://github.com/spring-projects/spring-boot/blob/2c5398898b9dc4843071d67113b4cbf048f5b99f/spring-boot-project/spring-boot-tools/spring-boot-maven-plugin/src/main/java/org/springframework/boot/maven/AbstractRunMojo.java#L202) example. Also javadoc explaining that it was replaced by `pluginApiMinVerison` would be nice. 
It's not very important since we'll fail anyway but at least it becomes clear that this is not supposed to be used anymore, for example if the properties are listed by another maven plugin.

Reply:
done